### ISSUE TITLE
pod spec line fails with Xcode 11 beta installed

### ISSUE THREAD

**Issue by colinhumber** [CocoaPods/fourflusher#21]

**Comment by colinhumber** [comment]
```
ERROR | [iOS] unknown: Encountered an unknown error (Could not find a `ios` simulator (valid values: ). Ensure that Xcode -> Window -> Devices has at least one `ios` simulator listed or otherwise add one.) during validation.
```

This message is thrown when running a `pod spec lint` after installing Xcode 11 beta.

**Comment by chrigu1981** [comment]
same here. thx in advance

**Comment by Coeur** [comment]
Fixed by #22.
